pcdsdevices/lakeshore.py

```diff
@@ -0,0 +1,87 @@
+"""
+Classes for lakeshore temperature controller
+"""
+import logging
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+from ophyd import FormattedComponent as FCpt
+
+from .interface import BaseInterface
+
+logger = logging.getLogger(__name__)
+
+
+class Heater(BaseInterface, Device):
+    """
+    HeaterState Channel Object.
+
+    Parameters
+    ----------
+    channel_prefix : str
+        The EPICS base of the HeaterState Channel. E.g.: ``
+    """
+    htr_state = FCpt(EpicsSignal, '{prefix}:GET_RANGE_{channel}', write_pv='{prefix}:PUT_RANGE_{channel}', kind='normal', doc='heater range')
+    htr_state_rbv = FCpt(EpicsSignalRO, '{prefix}:GET_HTRSTAT_{channel}', kind='normal')
+
+    # insert tab white list
```

Comment:
lingering todo comment, you probably want `tab_component_names = True` on all three classes (I think that's the invocation?)